app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -671,132 +671,140 @@ export const ContentForm: React.FC<IContentFormProps> = ({
                     />
                   </Show>
                 </Row>
-                <Row className="submit-buttons">
-                  <Row
-                    flex="1 1 0"
-                    className={contentType !== ContentTypeName.Image ? 'multi-section' : ''}
-                  >
-                    <Show visible={contentType === ContentTypeName.Snippet}>
-                      <Row className="multi-group">
-                        <TimeLogSection prepTimeRequired={true} />
-                      </Row>
-                    </Show>
-                  </Row>
-                  <Show
-                    visible={
-                      contentType === ContentTypeName.Snippet &&
-                      props.values.fileReferences.length === 0 &&
-                      !props.values.file
-                    }
-                  >
-                    <FormikCheckbox
-                      name="allowPublishWithoutFile"
-                      label="Allow publish without file"
-                      className="allow-no-file"
-                      value={allowPublishWithoutFile}
-                      checked={allowPublishWithoutFile}
-                      onChange={(e: any) => {
-                        setAllowPublishWithoutFile(e.target.checked);
-                      }}
-                    />
-                  </Show>
-                  <Button
-                    type="submit"
-                    disabled={
-                      props.isSubmitting ||
-                      ([ContentTypeName.Snippet, ContentTypeName.Image].includes(contentType) &&
-                        !allowPublishWithoutFile &&
-                        props.values.fileReferences.length === 0 &&
-                        !props.values.file)
-                    }
-                    onClick={() => {
-                      setSavePressed(true);
-                    }}
-                  >
-                    Publish
-                  </Button>
-                  <Show
-                    visible={
-                      props.values.status === ContentStatusName.Publish ||
-                      props.values.status === ContentStatusName.Published
-                    }
-                  >
-                    <Button
-                      variant={ButtonVariant.secondary}
-                      disabled={
-                        props.isSubmitting ||
-                        (contentType === ContentTypeName.Snippet &&
-                          props.values.fileReferences.length === 0 &&
-                          !props.values.file)
+                <Row gap="0.5rem">
```

Comment:
I've moved a lot of the components around.  Not 100% certain if all the logic was maintained correctly.